content/troubleshooting/accessing-builder-machine-via-ssh.md

```diff
@@ -64,8 +63,8 @@ If you have finished debugging the build, don't forget to end the session to avo
 
 * You can access your project on the builder machine by running `cd $FCI_BUILD_DIR`
 * If you encounter an error during the build, a good way to get started is by digging into the build logs on Codemagic and rerunning the first failed command.
-* The command `printenv` prints all the environment variables exported during the build. See [environment variables](../building/environment-variables/) for an explanation of read-only environment variables.
-* The `sudo` command is available to execute any commands with root privileges.
+* The command `printenv` prints all the environment variables exported during the build on Linux/macOS. Use `dir env:` on Windows to achieve the same in Powershell. See [environment variables](../building/environment-variables/) for an explanation of read-only environment variables.
+* The `sudo` command is available to execute any commands with root privileges on Linux/macOS. PowerShell on Windows machines is already running in privileged mode.
```

Comment:
the first four tips are just repeating what is already said just a few paragraphs above, should we have twice?